pkg/kubelet/container/runtime_cache.go

```diff
@@ -60,8 +60,8 @@ type runtimeCache struct {
 	updatingThreadStopTime time.Time
 }
```

Comment:
pleas add a comment describing the working of the runtimecache, either here or above NewRuntimeCache (high level is enough, something like: timstamp will be as new as the list, timestamps never move forward, we don't expect callers to modify pods in the pods list (this has to be true right?)). 

If you could include the rationale behind the udpatingThread (I think there was a dicsussion on some other pr?) that would be even better.